solutions/src/main/java/ru/sberbank/school/task06/MapCounter.java

```diff
@@ -0,0 +1,67 @@
+package ru.sberbank.school.task06;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Mart
+ * 25.05.2019
+ **/
+public class MapCounter <K> implements CountMap {
```

Comment:
GenericWhitespace &#39;&lt;&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-132926148>*